mesonbuild/backend/ninjabackend.py

```diff
@@ -1948,8 +1948,6 @@ def generate_rust_target(self, target: build.BuildTarget) -> None:
         args += ['--emit', f'dep-info={depfile}', '--emit', f'link={target_name}']
```

Comment:
`s/benefice/benefit/` in the commit message.

Reply:
thx fixed